### DISCUSSION TITLE
请问如何在ARM架构且OS系统为openeuler的cpu环境下安装该项目

### DISCUSSION THREAD

**Discussion by Puck00** [triton-lang/triton-cpu#167] [Q&A]
你好，我最近尝试按照当前项目readme中的安装指南在上述环境下编译该项目，发现编译过程中存在诸多问题，请问是否可以提供在上述环境下的更为详细的安装指南。
十分感谢。

**Comment by minjang** [comment]
> How to install this project on a cpu with ARM architecture and openeuler OS #167
>
> Hello, I recently tried to compile this project in the above environment according to the installation guide in the current project readme, and found that there are a lot of problems during the compilation process, could you please provide a more detailed installation guide in the above environment.
Thank you very much.

Hello, please write in English.

We only tested on a few limited ARM machines. We never tested on OpenEuler OS.

And, TritonCPU isn't different when it comes to the installation. Please double check whether you can install Triton on your environment: https://github.com/triton-lang/triton

**Reply by Puck00** [reply]
Thank you very much for your answer!

I've tried installing Triton via the instructions and source code in https://github.com/triton-lang/triton, but this method still fails in my environment.

Can you tell me the specific environment configuration under the ARM architecture you're trying to experiment with, such as the OS version, environment variables, etc.?

**Reply by minjang** [reply]
We have tested on GH200 with ARM Neoverse V2 CPU. OS is Centos 8.

**Reply by Puck00** [reply]
Thank you. 

I encounter a lot of compilation errors. What version of compiler are you using?

And I want to know what version of glibc are you using?
